pkgs/applications/networking/cluster/timoni/update.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=./. -i bash -p curl gnused nix jq
```

Comment:
Just a small thing nixpkgs manual states:
```
Note

An update script will be usually run from the root of the Nixpkgs repository but you should not rely on that. Also note that update.nix executes update scripts in parallel by default so you should avoid running git commit or any other commands that cannot handle that. 
```

becasue you are doing `nixpkg=./.` this dosn't work here, the fix for that would be using the path `git rev-parse --show-toplevel` provides

Reply:
Sorry, I'm a little bit new to this. How to test running it outside of the root properly so it does break?

Thanks a lot.

Reply:
You have to be inside the nixpkgs repo, when running the update script, but you shouldn't need to be in the root. You can for example just cd into nixos and try running the script with ../pkgs/.../timoni/update.sh which should also work but currently doesn't

> I'm a little bit new to this.

Don't worry, feel free to ask questions :D

Reply:
Thanks a lot. 

- forced pushed another fix
- also removed version check (because if there was some error, it could end in a broken state when the version was already updated, but the hashes we're not)